easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.6.3-iccifort-2011.13.367.eb

```diff
@@ -20,14 +20,11 @@ dependencies = [('hwloc', '1.6')]
 # needed for --with-openib
 osdependencies = [('libibverbs-dev', 'libibverbs-devel')]
 
+libs = ["mca_common_sm", "mpi_cxx", "mpi_f77", "mpi_f90", "mpi", "ompitrace", "open-pal",
+        "otfaux", "otf", "open-rte", "vt", "vt-hyb", "vt-mpi", "vt-mpi-unify"]
 sanity_check_paths = {
     'files': ["bin/%s" % binfile for binfile in ["ompi_info", "opal_wrapper", "orterun"]] +
-             ["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in ["mca_common_sm", "mpi_cxx",
-                                                                         "mpi_f77", "mpi_f90",
-                                                                         "mpi", "ompitrace", "open-pal",
-                                                                         "otfaux", "otf",
-                                                                         "open-rte", "vt", "vt-hyb",
-                                                                         "vt-mpi", "vt-mpi-unify"]] +
+             ["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in libs],
              ["include/%s.h" % x for x in ["mpi-ext", "mpif-common", "mpif-config", "mpif",
```

Comment:
the main issue was that the list of names was way to the right, this line doesn't suffer from that